### PR TITLE
update to commands

### PR DESCRIPTION
**Pull request by ryestew**
fixes https://github.com/ethereum/remix-project/issues/88

### CODE REVIEW

**Comment by ryestew** [comment]
in progress - do not merge

**Comment by ryestew** [comment]
can be merged - if approved

**Comment by ryestew** [comment]
@Aniket-Engg - how does this update fit your issue https://github.com/ethereum/remix-project/issues/88.  I mean this PR - sort of obliterates many of the commands.  Which ones should be expanded upon?

File: docs/remix_commands.md [hunk 1]
```diff
@@ -3,91 +3,32 @@ Remix Commands
 
 In the console, you can run the commands listed below.  Once you start to type a command, there is *auto completion*.  These commands are using the following libraries:
 
-+ *ethers*: The [ethers.js](https://docs.ethers.io/ethers.js/html/getting-started.html) library is a compact and complete JavaScript library for Ethereum. 
++ *remix*: See the remix commands below that can be run from the console.
```
Comment (Aniket-Engg) [inline docs/remix_commands.md:6]: There should be one line about remix commands and then a link to quickly jump to remix commands section. Same for all.

**Review by Aniket-Engg — COMMENTED** [review]
Overall, I suggest to create a dedicated commands section for `swarm`, `ethers` and `web3` along with at least one working example below each command (See Assert lib documentation).

**Pull request closed without merging**